dessia_common/workflow/core.py

```diff
@@ -106,6 +107,65 @@ def copy(self, deep: bool = False, memo=None):
         else:
             copied_default_value = UNDEFINED
         return Variable(type_=self.type_, default_value=copied_default_value, name=self.name)
+    
+    def process_heterogeneous_sequence(self, schema, import_str_list):
```

Comment:
You do not use self here, but instead a  lot of schema, while having a lot of "isinstance" stuff, so I guess this should be a method of schema that gives import depending on the type of schema.

This goes for all your `process_*` methods here

Reply:
refactored !

done